portfolio/src/main/webapp/script.js

```diff
@@ -26,3 +26,21 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+
+/**
+ * Adds random fact about me to the page
+ */
+function addRandomFact() {
+  const facts =
+    ["I'm a fun guy",
+     'Obviously I love web development',
+     "It's just more questions you have to ask me in order for me to tell you more about myself",
+     "I just can't give you the whole spill"];
```

Comment:
nit - looks like [single quotes are the style for javascript strings](https://google.github.io/styleguide/jsguide.html#features-strings-use-single-quotes).

Reply:
Okay, I'll adjust this one.

Reply:
FYI: `if a string contains a single quote character, consider using a template string to avoid having to escape the quote.`

But no need to take action on this now. I'll resolve.